src/Compute/Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -648,7 +649,36 @@ async Task SimpleParameterSetExecuteCmdlet(IAsyncCmdlet asyncCmdlet)
                 LoadBalancerStrategy.IgnorePreExistingConfigCheck = false;
             }
 
-            //Guest Attestation Identity stuff
+            // TL default for Simple Param Set, no config object
+            if (!this.IsParameterBound(c => c.SecurityType)
+                && !this.IsParameterBound(c => c.ImageName)
+                && !this.IsParameterBound(c => c.ImageReferenceId)
+                && !this.IsParameterBound(c => c.SharedGalleryImageId))
```

Comment:
Haider is making changes to this parameter: [[Compute] SharedGalleryImageId bug fix for New-Azvmss, New-Azvm by haagha · Pull Request #23145 · Azure/azure-powershell (github.com)](https://github.com/Azure/azure-powershell/pull/23145)

Note to check w him

Reply:
my changes are in VirtualMachineScaleSetStrategy.cs